### ISSUE TITLE
Cannot read property '0' of null

### ISSUE THREAD

**Issue by Mathieuka** [jhen0409/react-native-debugger#349]
Hello everyone, 

i have installed the last version of react native debugger, the connection work correctely but when i want dive in the tree of component for verify the props, my application crash and throw me "cannot read property '0' of null".
I would like to point that i have not yet installed the connection of Redux in my App.

AppContainer > RNRedux > Provider > Connect(App)>..."CRASH HERE"...</Connect(App)>

React Native Debugger app version: v0.9.7
React Native version: "0.59.8"
Platform: IOS
Is real device of platform: simulator
Operating System: MacOS

![Capture d’écran 2019-05-19 à 23 02 09](https://user-images.githubusercontent.com/34446722/57988218-be1b1700-7a8b-11e9-87d8-76e9a0b0d5ad.png)
<img width="737" alt="Capture d’écran 2019-05-19 à 23 02 16" src="https://user-images.githubusercontent.com/34446722/57988220-c2473480-7a8b-11e9-9555-79ab51d91571.png">
<img width="457" alt="Capture d’écran 2019-05-19 à 23 02 26" src="https://user-images.githubusercontent.com/34446722/57988221-c5dabb80-7a8b-11e9-98df-0afa67afffab.png">

**Comment by szilardM** [comment]
I have the same issue. Did you find a solution?

**Comment by Mathieuka** [comment]
Yes it's fixed but i don't now really why, by thinking about it, try to change the first value you try to read in your tree structure with a value other than "null".

<img width="314" alt="Capture d’écran 2019-05-31 à 15 38 51" src="https://user-images.githubusercontent.com/34446722/58709425-6ef1b200-83ba-11e9-8a77-8186a99da70b.png">

**Comment by mpcen** [comment]
This is still happening for me. Fresh install on expo
